cluster_kwargs.yaml

```
@@ -66,3 +66,9 @@ test_work_stealing_on_straggling_worker:
 test_repeated_merge_spill:
   n_workers: 20
   worker_vm_types: [m6i.large]
+
+# For tests/workflows/test_from_csv_to_parquet.py
+from_csv_to_parquet_cluster:
+  n_workers: 30
```

Sorry @milesgranger I think we still want to explicitly set them with the default, to avoid mixing of instance types. Basically, the same as we do in the case of test_xgboost or embarrasingly_parallel. 
```suggestion
  n_workers: 30
  worker_vm_types: [m6i.xlarge]  # 4 CPU, 16 GiB (preferred default instance)
```